src/util/utils.js

```diff
@@ -21,8 +24,29 @@ const responseStatusHandler = (status, entity, id, url) => {
   }
 };
 
+const sendViolationMetrics = (validationErrors, dropped, metaTags) => {
+  const vTags = {
+    "Unplanned-Event": 0,
+    "Additional-Properties": 0,
+    "Datatype-Mismatch": 0,
+    "Required-Missing": 0,
+    "Unknown-Violation": 0,
```

Comment:
Can we use enums or constants as keys in this object ? The error type is this and can change in future, let's hardcode them as constants ?

Reply:
matching with violation-types
to support line `vTags[error.type] += 1;`

Reply:
This will be removed after two weeks.